app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+
+  def new
+
+  end
```

Comment:
Empty controller methods like this can be re-written in one line like this `def new; end`